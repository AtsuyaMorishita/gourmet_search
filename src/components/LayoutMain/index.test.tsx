import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { LayoutMain } from ".";

describe("LayoutMain", () => {
  it("render correctly", () => {
    const { container } = render(<LayoutMain>ああ</LayoutMain>);
    expect(container).toMatchSnapshot();
  });
});
