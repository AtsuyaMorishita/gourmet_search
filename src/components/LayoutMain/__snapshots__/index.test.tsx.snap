// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutMain render correctly 1`] = `
<div>
  <main
    class="max-w-[1000px] mx-auto mt-[80px]  mb-[100px]"
  >
    ああ
  </main>
</div>
`;

exports[`header render correctly 1`] = `
<div>
  <main
    class="max-w-[1000px] mx-auto mt-[80px]  mb-[100px]"
  />
</div>
`;
